projects/sandbox/vizapp/vizapp/plots/vt.py

```diff
@@ -0,0 +1,179 @@
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+if TYPE_CHECKING:
+    import bilby
+
+import logging
+
+from bokeh.layouts import column, row
+from bokeh.models import (
+    Button,
+    ColumnDataSource,
+    LinearAxis,
+    NumericInput,
+    Range1d,
+)
+from bokeh.plotting import figure
+from vizapp.priors import gaussian_masses
+
+from bbhnet.analysis.vt import VolumeTimeIntegral
+
+MPC3_TO_GPC3 = 1e-9
+
+
+class VolumeTimeVsFAR:
+    def __init__(
+        self, height, width, source_prior: "bilby.core.prior.PriorDict"
+    ):
+        self.height = height
+        self.width = width
+        self.source_prior = source_prior
+        self.keys = self.source_prior.keys()
+
+        self.fars = np.logspace(0, 7, 7)
+        self.configure_sources()
+        self.configure_widgets()
+        self.configure_plots()
+        self.logger = logging.getLogger("VT vs FAR")
+        self.logger.debug(self.fars)
+
+    def configure_plots(
+        self,
```

Comment:
really minor but if you get rid of the comma black will let you keep this all on the same line